symplyphysics/laws/dynamics/springs/compliance_of_two_serial_springs.py

```diff
@@ -27,7 +32,35 @@
 
 law = Eq(total_compliance, first_compliance + second_compliance)
 
-# TODO: derive law from Hooke's law
+# Derive law from Hooke's law
+## In the case of parallel connection, the force acting on both springs are equal whereas
+## their deformations add up.
+
+_force = SymSymbol("force")
+
+_stiffness_expr = solve(compliance_def.definition, compliance_def.stiffness)[0]
+_first_stiffness = _stiffness_expr.subs(compliance_def.compliance, first_compliance)
+_second_stiffness = _stiffness_expr.subs(compliance_def.compliance, second_compliance)
+_total_stiffness = _stiffness_expr.subs(compliance_def.compliance, total_compliance)
+
+_deformation_expr = solve(
+    hookes_law.law, hookes_law.deformation
+)[0].subs(
+    hookes_law.spring_reaction, _force,
```

Comment:
Do we have any way to justify that force is the same on both springs?

Reply:
Hm, yes, using 3rd Newton's law of motion, for instance. I'll add it